exercises/go-counting/.meta/src/reference/java/GoCounting.java

```diff
@@ -0,0 +1,157 @@
+import java.awt.Point;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class GoCounting {
```

Comment:
This can be [package private](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html) :)